### DISCUSSION TITLE
Github pages 的顯示都跑掉了是什麼原因呢？

### DISCUSSION THREAD

**Discussion by sinyawww** [hexojs/hexo#5187] [Question / Answer] [answered]
連結：
https://github.com/sinyawww/titus-audio.git
https://sinyawww.github.io/titus-audio/

我在 VScode 裡面看是正常的呀，不知道為什麼上傳上去之後就好像 js 和資料夾的路徑都跑掉一樣 QQ 
真的不知道問題出在哪，還請各位指教，感謝 !

**Comment by stevenjoezhang** [comment, marked as answer]
Your webpage tries to load
```
https://sinyawww.github.io/titus-audio/universal/scripts-compressed/common-8a5ca6510bf9215598ae4-min.en-US.js
```

by the actual link to js file is
```
https://sinyawww.github.io/titus-audio/universal/scripts-compressed/common-8a5ca6510bf9215598ae4-min.en-us.js
```

I guess you have to configure git to make the file name uploaded case sensitive

**Reply by sinyawww** [reply]
OMG it works! Thank you so so much I can't find it by myself in my entire life. You really pinpoint the problem even I typed in Chinese :)
